business_logic/db/experimentsCollection.js

```diff
@@ -26,23 +26,53 @@ var experimentsCollection_global = null
 
 async function loadExperimentsCollection(database) {
   try {
-    if (!experimentsCollection_global){
+    if (!experimentsCollection_global) {
       experimentsCollection_global = await database.collection('Experiments');
     }
   }
   catch {
-      return null;
+    return null;
   }
 }
 
 
-  //delete the last experiment and insert the new one
-  function insertExperiment (experiment){
-    collection.remove({});
-    collection.insert(experiment);
-  }
+//delete the last experiment and insert the new one
+function insertExperiment(experiment) {
+  collection.remove({});
+  collection.insert(experiment);
+}
+function getExperiment(expId) {
+  return experimentsCollection.find({ exp_id: expId });
+}
+//remove after hackhton
+function getExperiments() {
+  return collection.find({});
+}
+
+function getExperimentByCode(expCode) {
+  return collection.find({ exp_code: expCode });
+}
+
+
+function insertParticipant(expId, participant) {
+  var username = participant.participant_twitter_username;
+  var groupId = participant.group_id;
+  var experiment = collection.find({ exp_id: expId });
+  var groups = experiment.exp_groups;
```

Comment:
It is better to use "let" instead of var inside a function (prevent problems that are hard to debug)